### DISCUSSION TITLE
remix项目国际化i18已经成功注入了，但是antd样式还是默认英文

### DISCUSSION THREAD

**Discussion by sewth** [ant-design/ant-design#45448] [Q&A]
```typescript
import { useTranslation } from 'react-i18next';
let { t } = useTranslation();
header {t('greeting')}
``` 
我在页面上试了下，greeting能正常显示你好。不过antd的组件还是默认英文。
<img width="305" alt="image" src="https://github.com/yyong008/remix-antd-admin-electron/assets/21299462/5f1f2dee-5096-4e27-b597-eeb907948c53">
<img width="305" alt="image" src="https://github.com/yyong008/remix-antd-admin-electron/assets/21299462/dc828a3b-abeb-4ec0-8c22-f0fb5b818901">

项目git地址是： https://github.com/sewth/remix-express-antd

请教下，是哪里还是有问题吗。

项目技术是
remix 2.0.1
express 4.18.2
antd: 5.9.4

**Comment by zombieJ** [comment]
参考 https://ant-design.antgroup.com/components/config-provider-cn#config-provider-demo-locale

**Comment by sewth** [comment]
> 参考 https://ant-design.antgroup.com/components/config-provider-cn#config-provider-demo-locale

ConfigProvider 我试了在entry.client.tsx entry.server.tsx root.tsx配置都不起作用，在SSR渲染里面好像不生效
